lighthouse-docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM node:9-slim
+
+LABEL maintainer="Eric Bidelman <ebidel@>"
+
+# Install utilities
+RUN apt-get update --fix-missing && apt-get -y upgrade
+
+# Install latest chrome dev package and fonts to support major charsets
+# (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-unstable
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont \
+      --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get purge --auto-remove -y curl \
+    && rm -rf /src/*.deb
+
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.1/dumb-init_1.2.1_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+
+# Download latest Lighthouse from npm.
+# Cache bust so we always get the latest version of LH when building the image.
+ARG CACHEBUST=1
+RUN npm i lighthouse -g
```

Comment:
lets go for yarn, it's installed on these images anyway
`RUN yarn global add lighthouse`

Reply:
I'm not sure it matters. Users aren't running this code.

Reply:
no not really but it might speedup the docker build if we already have a yarn.lock file. We use yarn everywhere so I thought why not use it in docker as well.